mlf_core/create/template_creator.py

```diff
@@ -278,16 +282,17 @@ def create_common_files(self) -> None:
                                     'license': self.creator_ctx.license,
                                     'project_short_description': self.creator_ctx.project_short_description,
                                     'github_username': self.creator_ctx.github_username,
-                                    'creator_github_username': self.creator_ctx.creator_github_username},
+                                    'creator_github_username': self.creator_ctx.creator_github_username,
+                                    'framework': self.creator_ctx.language.capitalize() if domain == 'mlflow' else ''},
                      no_input=True,
                      overwrite_if_exists=True)
 
-        # recursively copy the common files directory content to the created project
-        copy_tree(f'{os.getcwd()}/common_files_util', f'{cwd_project}/{self.creator_ctx.project_slug}')
+        # copy the common files directory content to the created project
+        copy_tree(f'{Path.cwd()}/common_{domain}_files', f'{cwd_project}/{self.creator_ctx.project_slug}')
         # delete the tmp cookiecuttered common files directory
-        delete_dir_tree(Path(f'{Path.cwd()}/common_files_util'))
+        delete_dir_tree(Path(f'{Path.cwd()}/common_{domain}_files'))
```

Comment:
I prefer this naming scheme, but open to any suggestions ;)

Reply:
fine with me